estate/models/estate_property.py

```diff
@@ -57,4 +57,27 @@ class EstateProperty(models.Model):
     offer_ids = fields.One2many(
         "estate.property.offer",
         "property_id",
+        " ",
     )
+    total_area = fields.Float(compute="_compute_total")
+    best_price = fields.Float("Best Offer", compute="_compute_best_price")
+
+    @api.depends("living_area", "garden_area")
+    def _compute_total(self):
+        for record in self:
+            record.total_area = record.living_area + record.garden_area
+
+    @api.depends("offer_ids.price")
+    def _compute_best_price(self):
+        for record in self:
+            prices = record.offer_ids.mapped("price")
+            record.best_price = max(prices) if prices else 0.0
```

Comment:
Could be a one-liner. Also, you could take advantage of kwarg `default` of `max` method